substrate/frame/conviction-voting/src/lib.rs

```diff
@@ -487,10 +496,36 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 								);
 								prior.accumulate(unlock_at, balance)
 							}
+						} else if v.1.as_standard() == Some(!approved) {
```

Comment:
`locked_if` should be changed to take an enum

```
enum LockedIf {
     Status(bool),
     Always,
}
```

`Always` would only be set when the `balance_locked_on_unsuccessful_vote` returns so.